### DISCUSSION TITLE
Prevent drop/paste images from computer.

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#5056] [Q&A] [answered]
Hello Arthur!

I have just discovered that we can drop files from computer or paste it inside a text file and it converts into base64 images.
Since We prevent to users to have images in base64, I would like to know if we have a property to prevent this to happen.

I'm looking in the code to see if there's anything that might help or for the configuration I'm looking for.

But the way I'm imagining doing it is using the component:add event and getting the image, but that's not user friendly

https://www.loom.com/share/f2f91e20a1f545528e007f2e22d2789d

Thank you Arthur.

**Comment by ronaldohoch** [comment]
The config:
```js
        assetManager:{
            dropzone:false
        },
```

Even with the above config, it continues to drop images from the computer to GrapesJs

**Comment by artf** [comment, marked as answer]
There is [an option](https://github.com/GrapesJS/grapesjs/blob/cb04b97cf6c6cd76676eb3aa095d5de54ef7bd1d/src/asset_manager/config/config.ts#L86) in assetManager that allows you to disable the embed of assets. If you have properly configured the [asset upload endpoint](https://grapesjs.com/docs/modules/Assets.html#uploading-assets), that will all allow to upload images to your backend when the user drops images in the canvas.

**Reply by ronaldohoch** [reply]
Thank you!
